.github/workflows/release_package.yml

```diff
@@ -0,0 +1,112 @@
+name: Publish Release
+
+on:
+  workflow_call:
+    inputs:
+      package-name:
+        required: true
+        type: string
+      package-org:
+        required: true
+        type: string
+      additional-build-flags:
+        required: false
+        type: string
+      additional-publish-flags:
+        required: false
+        type: string
+
+jobs:
+  publish-release:
+    name: Release Package
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'ballerina-platform'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: "temurin"
+          java-version: 11
```

Comment:
Wouldnt this checkout `ballerina-standard-library` instead of the required module?

Reply:
No, when the `actions/checkout` is used in a called workflow, it will be run as if it is invoked in the caller workflow.

Refer the `Overview` section here: https://docs.github.com/en/actions/using-workflows/reusing-workflows#overview

Reply:
Ack